registration/views/staff.py

```diff
@@ -164,12 +166,14 @@ def add_new_staff(request):
 def staff_index(request, guid):
     event = Event.objects.get(default=True)
     tz = timezone.get_current_timezone()
-    today = tz.localize(datetime.now())
+    today = datetime.now(python_tz.utc)
```

Comment:
I'm curious about the rationale for a change to UTC here, since the Django configuration is timezone-aware.  When for checking when to open forms or considering birthdays, we will be dealing with date objects with timezones set to the server's timezone.  The Django admin informs clients when the client is in a different timezone when editing a datetime  entry.

Reply:
If I am reading the docs correctly, that is just where the timezone is stored.  It isn't necessarily GMT.

Reply:
Since we have `USE_TZ = True` in our config, using `django.utils.timezone` for datetime operations will use Django's timezone aware date objects, Django will handle the correct timezone application according to the configuration.   If you're using the container, you can set the `TZ` environment variable to match your local timezone (e.g. `TZ=America/New_York`).  Generally when dealing with dates and time, we should use the Django functions to make sure the timezone setting is respected.

https://github.com/furthemore/APIS/blob/324bf13872fcca207805bd15c6ee763ceebf1a83/fm_eventmanager/settings.py.docker#L241

https://docs.djangoproject.com/en/5.1/topics/i18n/timezones/#naive-and-aware-datetime-objects